Chinese/推荐美剧/Breaking.Bad.md

```diff
@@ -0,0 +1,2 @@
+[Breaking Bad](https://en.wikipedia.org/wiki/Breaking_Bad), 又名《绝命毒师》，剧情主要内容是讲述一位在某中学教书的天才化学家得了肺癌，并且仅有数月的存活时间，偶然的机会他接触到制造毒品(drug, meth，冰毒)可以大赚一笔钱。于是，他打算运用自己
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.